lib/modules/showImages.js

```diff
@@ -260,6 +260,11 @@ module.options = {
 		value: true,
 		description: 'Autoplay inline videos',
 	},
+	keepNativeExpando: {
```

Comment:
Yeah I originally added those, but I was wrong, they're not actually identical: for example, reddit's embeds don't get added to history when you expand them (if you have that option enabled).

So I think it's fine to just override everything